src/LoginWithAmazon.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Options variable to request for implicit grant. Logic for assigning 'deviceSerialNumber' needs to be revisited.
+const options = {
+    scope : ['alexa:all', 'profile'],
+    scope_data : {"alexa:all":{"productID":"Silent_Alexa","productInstanceAttributes":{"deviceSerialNumber":"12345"}}}
+};
+export default class LoginWithAmazon extends React.Component {
+  render() {
+    return (
+        <button type='button' onClick={() => this.handleLogin()}><img alt="demoImage" src="https://images-na.ssl-images-amazon.com/images/G/01/lwa/btnLWA_gold_156x32.png"
```

Comment:
Format. Image is a different element nested inside button. Format it as such. 
Install MS Visual Studio so you don't need to worry about formatting.